modules/social_features/social_group/config/install/social_group.settings.yml

```diff
@@ -2,3 +2,7 @@ allow_group_create: true
 allow_group_selection_in_node: false
 address_visibility_settings: false
 default_hero: null
+cross_posting:
+  status: false
+  content_types: {  }
+  group_types_disabled: {  }
```

Comment:
To be consistent and inclusive I would propose use just `group_types` variable name, title and logic in all places.